### DISCUSSION TITLE
[Feature] Share pictures to other users not only by link

### DISCUSSION THREAD

**Discussion by wavesswe** [immich-app/immich#3454] [Feature Request]
### The feature

Be able to share pictures to other users so they show up in thier flow, so they can use them in albums and so on.

Similar to how you pic users in album share.

Now only sharable via link.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by whitephoenix117** [comment]
Would also like to see this feature; I wonder what is the point of other user accounts if not to share albums/ images
